snuba/datasets/plans/query_plan.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import annotations
+
+from abc import ABC
+from dataclasses import dataclass
+from typing import Callable, Sequence
+
+from snuba.query import RawQueryResult
+from snuba.query.query_processor import QueryProcessor
+from snuba.request import Request
+
+
+SingleQueryRunner = Callable[[Request], RawQueryResult]
```

Comment:
What does "Single" mean here (or in other words, why isn't this just called "QueryRunner"?)